### PR TITLE
(IAC-725) docs: Update SingleStore.md

### PR DESCRIPTION
**Pull request by supear**
Hi Thomas! Here are some edits for this new doc.

### CODE REVIEW

File: docs/user/SingleStore.md [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+
+After running viya4-deployment with the setting `DEPLOY=false` in your ansible-vars.yaml file, locate the `sas-bases/` directory, which is a peer to the `site-config/` directory underneath your SAS deployment's <base_dir>.
+
+Complete each step under the **SingleStore Cluster Definition** heading in the `sas-bases/examples/sas-singlestore/README.md` file in order to configure your SAS Viya with SingleStore deployment, noting the following exceptions:
 
-After running SAS Viya Deployment with the setting DEPLOY=false in your ansible-vars.yaml file, locate the `sas-bases/` folder which is a peer to the `site-config/` folder underneath your deployments <base_dir>.
+- Add a new Step 1a after Step 1:
 
-Follow each step under the `SingleStore Cluster Definition` heading in the `sas-bases/examples/sas-singlestore/README.md` file to configure your SingleStore deployment noting the exceptions below:
+>>1a. Copy `$deploy/sas-bases/components/sas-singlestore` into the `$deploy/site-config/sas-singlestore/component` directory.
```
Comment (thpang) [inline docs/user/SingleStore.md:25]: I used `components` as my directory vs `component` seemed to work. Are we sure each product will always have one component?

**Review by thpang — COMMENTED** [review]
Small comment otherwise looks good.

File: docs/user/SingleStore.md (hunk 1 above)
Comment (supear) [reply docs/user/SingleStore.md:25]: Yes, I think "`$deploy/site-config/sas-singlestore/component` directory" is better.

**Review by dhoucgitter — APPROVED** [review]
suggested updates look good to me

File: docs/user/SingleStore.md [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
 
-## Requirements for a SingleStore SAS Viya deployment
+## Requirements for SAS Viya with SingleStore
 
-If your SAS software order included SAS Viya with SingleStore, there are additional requirements that may apply to your deployment. The SAS Viya _IT Operations Guide_ provides detailed information about requirements for a SingleStore enabled deployment of SAS Viya. Access it [here](https://documentation.sas.com/doc/en/itopscdc/default/itopssr/n0jq6u1duu7sqnn13cwzecyt475u.htm#n0qs42c42o8jjzn12ib4276fk7pb)
+If your SAS software order included SAS Viya with SingleStore, additional requirements apply to your deployment. The SAS Viya _IT Operations Guide_ provides detailed information about requirements for a SingleStore-enabled deployment of SAS Viya. You can access the guide [here](https://documentation.sas.com/doc/en/itopscdc/default/itopssr/n0jq6u1duu7sqnn13cwzecyt475u.htm#n0qs42c42o8jjzn12ib4276fk7pb).
```
Comment (sayeun) [inline docs/user/SingleStore.md:7]: @supear - The documentation.sas.com/doc/en/... went to an internal page looked like.  Shouldn't we use this link instead? https://go.documentation.sas.com/doc/en/itopscdc/default/itopssr/n0jq6u1duu7sqnn13cwzecyt475u.htm#n0qs42c42o8jjzn12ib4276fk7pb

Comment (supear) [reply docs/user/SingleStore.md:7]: Let's use the "transformed" version of this URL, which removes the "en" locale: 
https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=itopssr&docsetTarget=n0jq6u1duu7sqnn13cwzecyt475u.htm#n0qs42c42o8jjzn12ib4276fk7pb

Comment (thpang) [reply docs/user/SingleStore.md]: Is this acceptable @sayeun

**Review by sayeun — APPROVED** [review]
The canonical URL works.  Thanks Susan.

**Review by thpang — APPROVED** [review]
LGTM

**Pull request merged**